MIGRATION.md

```diff
@@ -715,6 +715,56 @@ setup for source maps in Sentry and will not require you to match stack frame pa
 To see the new options, check out the docs at https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/,
 or look at the TypeScript type definitions of `withSentryConfig`.
 
+#### Updated the recommended way of calling `Sentry.init()`
+
+With version 8 of the SDK we will no longer support the use of `sentry.server.config.ts` and `sentry.edge.config.ts`
+files (note that `sentry.client.config.ts` is still supported and encouraged). Instead, please initialize the Sentry
```

Comment:
l: I think the note about the client config should be its own sentence and maybe bold. Users might skip over the sentence and think everything is now initialized in `instrument.ts`.